### PR TITLE
Trans single line

### PR DESCRIPTION
**Pull request by astroyouth**
99% confident all translations over multiple lines have been reduced to 1 line each

### CODE REVIEW

File: my-video-room/core/views/view-settings-core.php [hunk 1]
```diff
@@ -44,9 +44,7 @@
 		}
 		echo '<p>';
 		esc_html_e(
-			'The Following Settings define site wide video default parameters in case other modules have not set parameters there. These defaults will be used 
-			if a user has not selected a setting for the room configuration, or it hasnt been defined at the module level. You can use the Template Browser tab
-			to view room selection templates.',
+			'The Following Settings define site wide video default parameters in case other modules have not set parameters there. These defaults will be used if a user has not selected a setting for the room configuration, or it hasnt been defined at the module level. You can use the Template Browser tab to view room selection templates.',
```
Comment (alecsammon) [inline my-video-room/core/views/view-settings-core.php:47]: cane we remove the capitalisation from "Following Settings". Maybe put double quotes around "Template Browser" also to make it more clear it is a tab

Comment (astroyouth) [reply my-video-room/core/views/view-settings-core.php:47]: Done

File: my-video-room/library/class-shortcodedocuments.php [hunk 2]
```diff
@@ -48,11 +46,7 @@ public function render_general_shortcode_docs() {
...
 			<?php
 			echo esc_html__(
-				'Returns a Variety of useful Information about a room that you can place in your pages<br>
-				Room=(one of the following - meet-center, bookings-center, site-video-room) - selects the auto generated room type to query. This is required.<br>
-				Type (title) - Room Name (with spaces) - Type (slug) - returns the post slug (eg- www.c.com/jones has slug of Jones) - Type (post_id) - returns the WordPress Post ID of a room
-				Type (url) - returns URL of room. <BR>
-				Usage - <b>[getvideo_room_info room="bookings-center" type = "url"]</b> will return the URL of the Bookings Center]',
+				'Returns a Variety of useful Information about a room that you can place in your pages<br>Room=(one of the following - meet-center, bookings-center, site-video-room) - selects the auto generated room type to query. This is required.<br>Type (title) - Room Name (with spaces) - Type (slug) - returns the post slug (eg- www.c.com/jones has slug of Jones) - Type (post_id) - returns the WordPress Post ID of a room Type (url) - returns URL of room. <br>Usage - <b>[getvideo_room_info room="bookings-center" type = "url"]</b> will return the URL of the Bookings Center]',
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:49]: we should split this into multiple strings so the `<br>` can be taken out.
```
<?php esc_html_e('Returns ....'); ?><br />
<?php esc_html_r('Room=...'); ?><br />
....
```

Also - random capitalisation.
Also `www.c.com` should be taken out as you have done elsewhere

Comment (alecsammon) [reply my-video-room/library/class-shortcodedocuments.php:49]: Also - the shortcode should be coming from the constants (I I think is also wrong!)

File: my-video-room/library/class-shortcodedocuments.php [hunk 3]
```diff
@@ -86,14 +80,7 @@ public function render_buddypress_shortcode_docs() {
...
-					works with a Personal Video Room used in a non BuddyPress environment.
-					<p>There are no Guest Shortcodes needed, as normal room shortcodes work correctly for users who are signed out and thus
-					not in the BuddyPress loop. Normal meeting invites, links, guest reception settings are available for rooms whose hosts
-					enter via BuddyPress.',
+					'This Shortcode is designed to be used in BuddyPress profile pages. It is not available outside of the BuddyPress profile loop environment. It handles everything in the context of whose profile you are viewing. If you are viewing your own profile, then you get a host video experience, if you are looking at someone elses profile (or are signed out) then the guest page for that profile is rendered. The room that is rendered is the same as the Personal Video Room - and seamlessly works with a Personal Video Room used in a non BuddyPress environment. <p>There are no Guest Shortcodes needed, as normal room shortcodes work correctly for users who are signed out and thus not in the BuddyPress loop. Normal meeting invites, links, guest reception settings are available for rooms whose hosts enter via BuddyPress.',
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:83]: split into two sections - in this case two `<p>` might be better

File: my-video-room/library/class-shortcodedocuments.php [hunk 4]
```diff
@@ -227,11 +201,7 @@ public function render_personalmeeting_shortcode_docs() {
 			<td style="width:75%; text-align: left;">
 			<?php
 			esc_html_e(
-				'This shortcode will render only the <strong>settings </strong>page of the <strong>Host</strong>. This is useful if you just want to edit
-				the room settings without launching the full room. This shortcode determines its host from
-				the logged in user. If placed in anonymous/non-logged in areas of the site the shortcode will return blank.
-				<strong>Please note</strong> admin settings for personal rooms are shared between BuddyPress Profile Rooms and Personal Video Rooms as
-				they are effectively the same room, with multiple entrances ',
+				'This shortcode will render only the <strong>settings </strong>page of the <strong>Host</strong>. This is useful if you just want to edit the room settings without launching the full room. This shortcode determines its host from the logged in user. If placed in anonymous/non-logged in areas of the site the shortcode will return blank. <strong>Please note</strong> admin settings for personal rooms are shared between BuddyPress Profile Rooms and Personal Video Rooms as they are effectively the same room, with multiple entrances ',
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:204]: the space should be moved from inside the `<strong>` to after

File: my-video-room/library/class-shortcodedocuments.php (hunk 2 above)
Comment (astroyouth) [reply my-video-room/library/class-shortcodedocuments.php:49]: Done

File: my-video-room/library/class-shortcodedocuments.php (hunk 3 above)
Comment (astroyouth) [reply my-video-room/library/class-shortcodedocuments.php:83]: done

File: my-video-room/library/class-shortcodedocuments.php (hunk 4 above)
Comment (astroyouth) [reply my-video-room/library/class-shortcodedocuments.php:204]: Done

File: my-video-room/library/class-shortcodedocuments.php [hunk 5]
```diff
@@ -48,14 +46,24 @@ public function render_general_shortcode_docs() {
...
-				Type (url) - returns URL of room. <BR>
-				Usage - <b>[getvideo_room_info room="bookings-center" type = "url"]</b> will return the URL of the Bookings Center]',
+				'Returns a Variety of useful Information about a room that you can place in your pages',
 				'my-video-room'
 			);
 			?>
+			<br />
+			<?php
+				echo esc_html__(
+					'Room=(one of the following - meet-center, bookings-center, site-video-room) - selects the auto generated room type to query. This is required.<br>Type (title) - Room Name (with spaces) - Type (slug) - returns the post slug (eg- www.c.com/jones has slug of Jones) - Type (post_id) - returns the WordPress Post ID of a room Type (url) - returns URL of room. <br>Usage - ',
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:56]: can we use the parameter here for `www.c.com/jones` to match the other change you made

File: my-video-room/library/class-shortcodedocuments.php [hunk 6]
```diff
@@ -48,14 +46,24 @@ public function render_general_shortcode_docs() {
...
+					'my-video-room'
+				);
+			?>
+			<br />
+			<?php
+				echo esc_html__(
+					'<strong>[getvideo_room_info room="bookings-center" type = "url"]</strong> will return the URL of the Bookings Center]',
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:63]: this shortcode looks wrong - maybe should use a constant

File: my-video-room/library/class-shortcodedocuments.php [hunk 7]
```diff
@@ -84,14 +92,23 @@ public function render_buddypress_shortcode_docs() {
 				<tr>
 				<td style="width:25%; text-align: left;"><h2>[ccbpboardroomswitch]</h2></td>
 				<td style="width:75%; text-align: left;">
+				<p>
 				<?php
 				esc_html_e(
 					'This Shortcode is designed to be used in BuddyPress profile pages. It is not
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:98]: this one is not  fixed with the new lines

File: my-video-room/library/class-shortcodedocuments.php [hunk 8]
```diff
@@ -84,14 +92,23 @@ public function render_buddypress_shortcode_docs() {
...
-					<p>There are no Guest Shortcodes needed, as normal room shortcodes work correctly for users who are signed out and thus
+					',
+					'my-video-room'
+				);
+				?>
+				</p>
+				<p>
+				<?php
+				echo esc_html__(
+					'There are no Guest Shortcodes needed, as normal room shortcodes work correctly for users who are signed out and thus
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:111]: and this one also needs to be put onto a single line

File: my-video-room/library/class-shortcodedocuments.php (hunk 5 above)
Comment (astroyouth) [reply my-video-room/library/class-shortcodedocuments.php:56]: Done

File: my-video-room/library/class-shortcodedocuments.php (hunk 7 above)
Comment (astroyouth) [reply my-video-room/library/class-shortcodedocuments.php:98]: done

File: my-video-room/library/class-shortcodedocuments.php (hunk 8 above)
Comment (astroyouth) [reply my-video-room/library/class-shortcodedocuments.php:111]: Done

**Pull request merged**